roles/ceph-common/tasks/docker/fetch_image.yml

```diff
@@ -0,0 +1,27 @@
+---
+# Normal case - pull image from registry
+- name: pull ceph daemon image
+  shell: "docker pull {{ ceph_docker_username }}/{{ ceph_docker_imagename }}"
```

Comment:
For both of these, I didn't actually add or change the existing code at all, just moved it to a common module.  I could fix those up, but maybe that should be a different commit?

(Sorry for the delay, my mail was being filtered.)